internal/npm_install/test/golden/node_modules/@gregmagolan/test-a/BUILD.bazel.golden

```diff
@@ -12,7 +12,6 @@ filegroup(
 filegroup(
     name = "test-a__files",
     srcs = [
-        ":.bin/test",
```

Comment:
is this change intentional?

Reply:
That comes with the yarn update. It no longer creates those files.